docs/source/releases/latest/666-final-bug-fixes-prior-to-workshop.yaml

```diff
@@ -0,0 +1,37 @@
+documentation:
+- description: "Finalize README so installation process works correctly."
+  title: "Add create_plugin_registries and missing test data to README"
+  related-issue: |
+    *From GEOIPS/geoips#666: 2024-07-15, bug fixes prior to workshop*
+  files:
+    modified:
+    - "README.md"
```

Comment:
run build html is failing for this, I believe because the CI is not pip installing the current package during the build html docs job.  sphinx relies on the package being pip installed in order to autodoc the api, so I think build html docs CI needs to pip install the plugin package, not just geoips.

I would like to just approve and merge this, so we can fully test the installation from a clean main install, and we can resolve the CI for a future merge.

When I created latest.rst with brassy first, then ran build_docs.sh, it completed succesfully:

Return values:
  Python install: 
  sphinx html:    0
  sphinx latex:   None
  latex pdf make: None
  warnings html:  
  warnings latex: 
  warnings make:  

Final retval: 0

But the CI fails:
https://github.com/NRLMMD-GEOIPS/geoips_clavrx/actions/runs/10016119862/job/27688637222?pr=39

Reply:
https://github.com/NRLMMD-GEOIPS/geoips/pull/693

That is the temporary fix (just install from build_docs.sh), the "correct" fix to the CI will be forthcoming.

Reply:
This is now passing as expected. The long-term fix discussed in issue #696.